packages/vulkan_headers.rb

```diff
@@ -1,29 +1,29 @@
-# Adapted from Arch Linux vulkan-headers PKGBUILD at:
-# https://github.com/archlinux/svntogit-packages/raw/packages/vulkan-headers/trunk/PKGBUILD
-
 require 'package'
 
 class Vulkan_headers < Package
   description 'Vulkan header files'
   homepage 'https://github.com/KhronosGroup/Vulkan-Headers'
-  version '1.2.157'
+  version '1.2.165'
   compatibility 'all'
-  source_url 'https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.2.157.tar.gz'
-  source_sha256 'dbc121f58641acd45c386ee96ecd5e10a124c489087443d7367fff4b53b49283'
+  source_url 'https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.2.165.tar.gz'
+  source_sha256 '3f9435a93ba13d94d0c3265a47e0436579e46bb9ca085e9b16a753458e4d79d2'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.157-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.157-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.157-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.157-chromeos-x86_64.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.165-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.165-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.165-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vulkan_headers-1.2.165-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'b130e7ad0d764c3e829d0da62260ebdbe192deb699a1540eb9a347b2c1948429',
-     armv7l: 'b130e7ad0d764c3e829d0da62260ebdbe192deb699a1540eb9a347b2c1948429',
-       i686: '75afde70082d150fbf190381da67e6d179968c8d0ecd59151b57a2fade44e8dc',
-     x86_64: '6bc64672860d7c8dc181d730b892ade24003e28b86102be78df28a6025d390b6',
+     aarch64: '03d07ab7485d23e1c0018f2ec02c877997382db4abf3ae80a43cda6a494c5383',
+      armv7l: '03d07ab7485d23e1c0018f2ec02c877997382db4abf3ae80a43cda6a494c5383',
+        i686: '66ec66419c0725af304b422462364449785c92c2599f392044b31d39d4d4ce4b',
+      x86_64: 'f4a32c518e378b6ec9c6d7535df7b7269a04ee0bdff98c50435219fbf6610788',
   })
 
+
+
+
```

Comment:
Why so many extra blank lines?

Reply:
Artifact of script to add hashes from files.